crates/nu-command/src/math/min.rs

```diff
@@ -15,15 +15,15 @@ impl Command for SubCommand {
     fn signature(&self) -> Signature {
         Signature::build("math min")
             .input_output_types(vec![
-                (Type::List(Box::new(Type::Number)), Type::Number),
+                (Type::List(Box::new(Type::Any)), Type::Any),
```

Comment:
i see this is because the implementation of `math min` and `math max` allow any input but yeah it looks super weird on some occasions :thinking: 

maybe we should restrict a bit in the future :eyes:

Reply:
In theory we could list the more likely ones as well.